internal/gps/pkgtree/pkgtree.go

```diff
@@ -16,8 +16,13 @@ import (
 	"strconv"
 	"strings"
 	"unicode"
+
+	radix "github.com/armon/go-radix"
 )
 
+// recursive ignore suffix
+const recIgnoreSuffix = "/*"
```

Comment:
i don't see a reason why we actually have to restrict to immediately following a slash - i think our only criteria here is that the asterisk has to be the last character.